src/test/scala/com/microsoft/partnercatalyst/fortis/spark/SparkContextCoordinator.scala

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import org.apache.spark.{SparkConf, SparkContext}
+
+object SparkContextCoordinator {

```

Comment:
There is a project called [spark-testing](https://github.com/holdenk/spark-testing-base) which has a bunch of smart base classes for unit testing Spark apps. I looked into this in the past but unfortunately they don't have streaming support so I dropped the investigation. Given that we now have a few use-cases for Spark Streaming tests, I wonder if we should look into contributing a streaming test base class to their project. Worth discussing during standup and I'm happy to take the task if we decide to go down this route.